src/styles/blocks/card.scss

```diff
@@ -0,0 +1,125 @@
+.catalog_container {
```

Comment:
These styles should be in separate scss file, they're not related to card styles